### ISSUE TITLE
Allow SiteConfig to dictate how long articles stay in sidebar tags

### ISSUE THREAD

**Issue by leewynne** [forem/forem#6805]
I think it will benefit new communities that don't have a lot of initial content to ensure that some of the sidebar tags display content.

Ideally this would be set in SiteConfig, maybe with a single combined date picker.
<img width="302" alt="Screenshot 2020-03-24 at 09 14 16" src="https://user-images.githubusercontent.com/12694270/77408593-dea0b580-6daf-11ea-9200-dbf4a265506d.png">

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by Ridhwana** [comment]
NOTE: this should also apply to the stories section too

**Comment by cmgorton** [comment]
Closing this in favor of adding it as a discussion on forem.dev so we can get products eyes on it.  Let me know if you would like me to create a post over there. Thanks Lee!
